src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -162,6 +188,33 @@ public static function get_core_data() {
 		return static::$core;
 	}
 
+	/**
+	 * Checks whether the registered blocks are the same
+	 * as the ones we already processed for this origin.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param $origin One of 'core', 'theme'. Source of data we cache blocks for.
+	 *
+	 * @return boolean
```

Comment:
It's good if we have a description for the return type.

I.e. 
`* @return boolean True on success, false otherwise`